src/app/zap-templates/common/StructHelper.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
could we find more focused names that do no use helper/support/util/core naming?

Maybe 'StructAtrributes' or 'StructProperties' would match, since it seems we only expose a 'is this type a struct'.
Also could be 'StructType.js'.

Reading the PR more, I see we already  use this '*Helper.js' pattern. Maybe followup to rename them in a separate PR?

Reply:
I have opened https://github.com/project-chip/connectedhomeip/issues/8198 for renaming them. I would prefer to rename them in a followup, especially if this PR is going to be cherry-picked.